CorsixTH/Lua/dialogs/confirm_dialog.lua

```diff
@@ -67,11 +75,17 @@ function UIConfirmDialog:UIConfirmDialog(ui, text, callback_ok, callback_cancel)
     :setTooltip(_S.tooltip.window_general.confirm):setSound"YesX.wav"
 
   self:registerKeyHandlers()
+  if self.is_error then self:forcePause() end
 end
 
--- Confirm dialogs are used for errors and require the game to pause
+-- Confirm dialogs are used for errors, if it is an error then pause the game
 function UIConfirmDialog:mustPause()
-  return true
+  return self.is_error
+end
+
+-- Errors force pausing
```

Comment:
How is 'errors' relevant here? There is nothing about errors in the `forcePause` function.

Reply:
Ah I've mixed up the what and why. I'll change this later